ct.yaml

```diff
@@ -5,3 +5,4 @@ debug: true
 helm-extra-args: --timeout 120s
 charts:
   - charts/connect
+  - charts/kubernetes-secrets-injector
```

Comment:
Will it be ok if we only say `secrets-injector`? Helm is already used for Kubernetes, so saying it again here may not be the most useful.

Reply:
That makes sense.
Then maybe also rename the folder also? `charts/kubernetes-secrets-injector` to `charts/secrets-injector`?